### PR TITLE
On-demand feature for AWS pipeline addition to dataduct

### PR DESCRIPTION
**Pull request by kewlkiev**
On-demand feature addition to dataduct
https://github.com/coursera/dataduct/issues/237

Remove Schedule object from YAML.
Make Schedule pipeline object None.
Change default object's scheduleType to on-demand

### CODE REVIEW

**Comment by kewlkiev** [comment]
@idralyuk @everanurag can you please review the change?

File: dataduct/pipeline/schedule.py [hunk 1]
```diff
@@ -37,6 +37,7 @@
     '8-hours': ('8 hours', None),
     '12-hours': ('12 hours', None),
     'one-time': ('15 minutes', 1),
+    'on-demand': ('ondemand',None),
```
Comment (idralyuk) [inline dataduct/pipeline/schedule.py]: Needs a space between , and None

Comment (kewlkiev) [inline dataduct/pipeline/schedule.py]: Done

**Comment by idralyuk** [comment]
Cool, thank you.

**Pull request merged**